misc/e2e_tests/src/e2e_conduit/fixtures/processors/block_evaluator.py

```diff
@@ -0,0 +1,25 @@
+from e2e_conduit.fixtures.plugin_fixture import PluginFixture
+
+
+class BlockEvaluator(PluginFixture):
```

Comment:
Why doesn't `BlockEvaluator` need to implement `resolve_config_output()` ?

Reply:
In the most recent updates you can kind of see the vision hopefully. The idea is to accumulate config values that get passed along to subsequent plugins since there are some implicit dependencies. I've just used a dict for that which gets updated by the `config_output` whereas the `config_input` is what actually gets written to the conduit config file.

Let me know if you have suggestions/comments on how to better tackle that problem in python.